### DISCUSSION TITLE
Buffer size error

### DISCUSSION THREAD

**Discussion by cloud9-kj** [google/timesketch#1813] [Q&A, quick solutions, support]
Hi, 

This error message is keep coming up, is it possible to increase the size?

/usr/lib/python3.8/os.py:1023: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used return io.open(fd, *args, **kwargs)

Also, is there a way to increase the CSRF token time? Thanks

**Comment by kiddinn** [comment]
I don't see how this error message that you are displaying there has anything to do with the CSRF token time.

But the answer to that can be found (here)[https://github.com/google/timesketch/issues/1794]).

Regarding that error message you are seeing, please see the discussion about that (here)[https://bugs.python.org/issue32236]

under what condition did you see this error message?

**Reply by cloud9-kj** [reply]
This occurs when I upload a file to the server via Web UI
